HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -223,7 +223,7 @@ def customizeHLTfor38761(process):
      if 'hltSiPixelRecHitsSoA' in process.__dict__:
          process.hltSiPixelRecHitsSoA.cpu =  cms.EDAlias(hltSiPixelRecHitsFromLegacy = cms.VPSet(
             cms.PSet(
-                type = cms.string('pixelTopologyPhase1TrackingRecHit2DCPUT')
+                type = cms.string('pixelTopologyPhase1TrackingRecHitSoAHost')
```

Comment:
Picky, but please use a separate customisation function for this change (as opposed to modifying one from a previous PR), e.g.
```diff
diff --git a/HLTrigger/Configuration/python/customizeHLTforCMSSW.py b/HLTrigger/Configuration/python/customizeHLTforCMSSW.py
index 53b007a50b7..98fe290725c 100644
--- a/HLTrigger/Configuration/python/customizeHLTforCMSSW.py
+++ b/HLTrigger/Configuration/python/customizeHLTforCMSSW.py
@@ -210,6 +210,12 @@ def customiseForOffline(process):
 
     return process
 
+def customizeHLTfor40465(process):
+    try:
+        process.hltSiPixelRecHitsSoA.cpu.hltSiPixelRecHitsFromLegacy[0].type = 'pixelTopologyPhase1TrackingRecHitSoAHost'
+    except:
+        pass
+    return process
 
 # CMSSW version specific customizations
 def customizeHLTforCMSSW(process, menuType="GRun"):
@@ -219,4 +225,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     # add call to action function in proper order: newest last!
     # process = customiseFor12718(process)
 
+    process = customizeHLTfor40465(process)
+
     return process
```